### ISSUE TITLE
Same model, different outputs for Keras versions 1.0.5 and 1.1.0

### ISSUE THREAD

**Issue by emrcak** [fchollet/keras#4094]
While trying to load an earlier trained model and reproduce the outputs, I observed very different outputs for the same input and same model loaded in different Keras versions (namely 1.0.5 and 1.1.0). The model was trained with Keras 1.0.5. I tried to also verify this with a dummy input data fed to the same model loaded in different versions. I checked that the loaded weights match for two versions. Is there any difference between how the layers are defined for v_1.0.5 and v_1.1.0? I noticed at least the default parameters for batch normalization layer is slightly different. For what it's worth, I get similar accuracy when I train two models with same parameters with these two versions (about 1% change), but the difference when loading the model in the other version is huge. I attached the model parameters and summary output.

[model_summary.txt](https://github.com/fchollet/keras/files/533414/model_summary.txt)

**Comment by keunwoochoi** [comment]
BN was in Theano backend around there (97d2a73, 0588393). By the way, how did you get the model summary text file?
